scripts/override_repo.py

```diff
@@ -0,0 +1,111 @@
+#!/bin/python3
+
+import argparse
+import os
+import subprocess
+import shutil
+
+TEMPLATE_REPO_URL_KEY = "ASSIGNMENT_TEMPLATE_REPOSITORY"
+DEFAULT_TEMPLATE_REPO_URL = "https://github.com/akirsch1/d2g-dummy-template"
+
+DIR_PREFIX = "template/"
+
+DEFAULT_NO_OVERRIDE = [".git/"]
+
+def _print_usage():
+    print("Execute override_repo.py -h to print usage information.")
```

Comment:
Das klingt irgendwie lustig. Wenn ich `usage` aufrufe (vermutlich über falsche Argumente oder mit `-h`), kriege ich die Antwort, ich möge genau das nochmal tun :)

Reply:
Ja, schön gelöst ist es nicht. Ich könnte alternativ den ArgParser global machen und dann an dieser Stelle dessen Hilfetext direkt ausgeben.

Reply:
Ich habe das Erstellen des Argument Parsers jetzt in eine eigene Funktion ausgelagert. Damit wird direkt die richtige Hilfe ausgegeben.